docs/tutorials/average_optimizers_callback.ipynb

```diff
@@ -0,0 +1,660 @@
+        "                loss='sparse_categorical_crossentropy',\n",
+        "                metrics=['accuracy'])\n",
+        "\n",
+        "  return model"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "pwdM2pl3RSPb"
+      },
+      "source": [
+        "## Prepare Dataset"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab_type": "code",
+        "id": "mMOeXVmbdilM",
+        "colab": {}
+      },
+      "source": [
+        "#Load Fashion MNIST dataset\n",
+        "train, test = tf.keras.datasets.fashion_mnist.load_data()\n",
+        "\n",
+        "images, labels = train\n",
+        "images = images/255.0\n",
+        "labels = labels.astype(np.int32)\n",
+        "\n",
+        "fmnist_train_ds = tf.data.Dataset.from_tensor_slices((images, labels))\n",
+        "fmnist_train_ds = fmnist_train_ds.shuffle(5000).batch(32)\n",
+        "\n",
+        "test_images, test_labels = test"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "iEbhI_eajpJe",
+        "colab_type": "text"
+      },
+      "source": [
+        "We will be comparing three optimizers here:\n",
+        "\n",
+        "*   Unwrapped SGD\n",
+        "*   SGD with Moving Average\n",
+        "*   SGD with Stochastic Weight Averaging\n",
+        "\n",
+        "And see how they perform with the same model."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "_Q76K1fNk7Va",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "#Optimizers \n",
+        "sgd = tf.keras.optimizers.SGD(0.01)\n",
+        "moving_avg_sgd = tfa.optimizers.MovingAverage(sgd)\n",
+        "stocastic_avg_sgd = tfa.optimizers.SWA(sgd)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "nXlMX4p9qHwg",
+        "colab_type": "text"
+      },
+      "source": [
+        "Both ```MovingAverage``` and ```StocasticAverage``` optimers use ```ModelAverageCheckpoint```."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "SnvZjt34qEHY",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "#Callback \n",
+        "checkpoint_path = \"./training/cp-{epoch:04d}.ckpt\"\n",
+        "checkpoint_dir = os.path.dirname(checkpoint_path)\n",
+        "\n",
+        "cp_callback = tf.keras.callbacks.ModelCheckpoint(filepath=checkpoint_dir,\n",
+        "                                                 save_weights_only=True,\n",
+        "                                                 verbose=1)\n",
+        "avg_callback = tfa.callbacks.average_model_checkpoint.AverageModelCheckpoint(filepath=checkpoint_dir, \n",
```

Comment:
It seems this variable is never used. 

Could you also use the alias? `tfa.callbacks.AverageModelCheckpoint`

Reply:
I've added `load_weights` before evaluation to demonstrate the callback works fine.